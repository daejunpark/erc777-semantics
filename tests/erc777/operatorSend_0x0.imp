test.preamble();

// operatorSend from 0x0 - must revert.
// operatorSend to 0x0  - must revert.

test.makeCaller(9);           // switch to 9
test.operatorSend(0, 1, 0, "", "");  // transfer 0 from 0 to 1
test.operatorSend(1, 0, 1, "", "");  // transfer 1 from 1 to 0
