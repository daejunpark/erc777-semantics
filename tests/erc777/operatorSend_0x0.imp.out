Changing the caller to 9
9 is sending 0 from 0 to 1 (data:""),  (operatorData:"")
REVERT
Balance of 0 is 0
Balance of 1 is 10
9 is sending 1 from 1 to 0 (data:""),  (operatorData:"")
REVERT
Balance of 1 is 10
Balance of 0 is 0
